x-pack/metricbeat/module/aws/utils.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
@ruflin @jsoriano @sayden What's your opinion on this name `utils.go`? I want to have a file contain all the shared functions that will be used for almost all metricsets in AWS module.

Reply:
SGTM